tests/actions/PolicyMemberTest.ts

```diff
@@ -228,4 +228,40 @@ describe('actions/PolicyMember', () => {
             });
         });
     });
+
+    describe('addMembersToWorkspace', () => {
+        it('Add new member to a workspace', async () => {
```

Comment:
```suggestion
        it('Add a new member to a workspace whilst offline', async () => {
```

Reply:
I didn't add the offline word because optimistic data is not specific to offline test